src/redux/reducers/componentConfig/initialState.js

```diff
@@ -294,7 +294,7 @@ const embeddingPreviewMitochondrialContentInitialConfig = {
   colour: colourBaseState,
   marker: markerBaseState,
   label: labelBaseState,
-  selectedSample: 'All',
+  selectedSample: 'sample',
```

Comment:
🤔 ?

Reply:
So that it uses the same cell set as the categorical embedding.